client/src/app/queries/tasks.ts

```diff
@@ -84,6 +89,47 @@ export const useFetchTaskDashboard = (refetchDisabled: boolean = false) => {
   };
 };
 
+export const useInfiniteServerTasks = (
+  initialParams: HubRequestParams,
+  refetchInterval?: number
+) => {
+  return useInfiniteQuery({
```

Comment:
The form takes a bit to understand, but looks nice and consistent with the rest of the queries code.